staff/alex-barba/cv/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Àlex Barba - CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Àlex Barba <img src="myphoto.jpg" alt="àlex barba photo" height="60" width="42"></h1> 
+        <p>C/Mercat 2 Esc B 3 3, 08750 - Molins de Rei, BCN</p>
```

Comment:
it will be nice to use the tag `address` for the adress information

Reply:
https://www.w3schools.com/tags/tag_address.asp

Reply:
Thx! Didn't know about it. Highly appreciated!